docs/Push-Server-API.md

```diff
@@ -1246,6 +1246,7 @@ Update an iOS configuration.
     "bundle": "some.bundle.id",
     "keyId": "KEYID123456",
     "teamId": "TEAMID123456",
+    "environment": null,
```

Comment:
This could use a better example, or maybe additional documentation to summarize the choices.